androidApp/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,11 @@
 <resources>
-    <string name="app_name">Press</string>
+  <string name="app_name">Press</string>
+
+  <item name="homeViewId" type="id" />
+  <item name="editorViewId" type="id" />
```

Comment:
I believe this won't be needed if we remove manual state saving from `EditorView`.